src/services/handler/continue-message-handler.js

```diff
@@ -14,6 +22,39 @@ export default async function continueMessageHandler(message) {
 
   logInfo('Trying to handle continue request');
 
+  const registrationRequest = await getRegistrationRequestByConversationId(conversationId);
+
+  switch (registrationRequest?.status) {
+    case Status.SENT_FRAGMENTS:
+    case Status.EHR_INTEGRATED:
+      logWarning(`Ignoring duplicate continue request. Conversation ID ${conversationId} already completed successfully`);
+      break;
+
+    case Status.INCORRECT_ODS_CODE:
+    case Status.MISSING_FROM_REPO:
+    case Status.EHR_DOWNLOAD_FAILED:
+    case Status.CORE_SENDING_FAILED:
+    case Status.EHR_INTEGRATION_FAILED:
+      logWarning(`Ignoring duplicate continue request. Conversation ID ${conversationId} already failed and is unable to retry`);
+      break;

```

Comment:
A bunch of cases with no actions. Intentional?

Reply:
https://www.freecodecamp.org/news/fall-through-in-javascript-switch-statements/